addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties

```diff
@@ -218,15 +218,12 @@ ascanbeta.shellshock.ref = http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2
 ascanbeta.shellshock.soln = Update Bash on the server to the latest version
 ascanbeta.shellshock.timingbased.evidence = Using the attack, a delay of [{0}] milliseconds was induced and detected
 
-ascanbeta.sourcecodedisclosure.desc = The source code for the current page was disclosed by the web server
+ascanbeta.sourcecodedisclosure.desc = The source code for the current page was disclosed by the web server.
 ascanbeta.sourcecodedisclosure.gitbased.evidence = The source code for [{0}] was extracted using [{1}]
-ascanbeta.sourcecodedisclosure.gitbased.extrainfo = 
 ascanbeta.sourcecodedisclosure.gitbased.name = Source Code Disclosure - Git 
 ascanbeta.sourcecodedisclosure.gitbased.soln = Ensure that Git metadata files are not deployed to the web server or application server
-ascanbeta.sourcecodedisclosure.lfibased.evidence = 
```

Comment:
Setting this blank was pointless.

Reply:
It was pointless defining it, not being blank.